coq/coq-diffs.el

```diff
@@ -0,0 +1,48 @@
+(require 'coq-db)
+
+(defun insert-with-face (str face)
```

Comment:
@jfehrle 

Regarding the structure of your PR, given you create a new file `coq/coq-diffs.el` with these functions, I'd suggest:

* Running `M-x checkdoc` in the corresponding Elisp buffer to add missing sections (`Commentary:` etc.)
* Prefixing the new functions with `coq-` or so to avoid possible clashes with other packages